apps/server/src/config/config-module-options.ts

```diff
@@ -0,0 +1,17 @@
+import { ConfigModuleOptions } from '@nestjs/config';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const createConfigModuleOptions = (config: () => Record<string, any>): ConfigModuleOptions => {
+	const options = {
+		isGlobal: true,
+		validationOptions: { infer: true },
+		// hacky solution: the server config is loaded in the validate step.
```

Comment:
Should be: "// hacky solution: the config is loaded in the validate step." No more restricted to the server.